scanner/src/main/kotlin/ScanResultsStorage.kt

```diff
@@ -77,7 +77,7 @@ abstract class ScanResultsStorage : PackageBasedScanStorage {
             storage = when {
                 configuredStorages.isNullOrEmpty() -> createDefaultStorage()
                 configuredStorages.size == 1 -> createStorage(configuredStorages.values.first())
-                else -> createCompositeStorage(config)
+                else -> NoStorage()
```

Comment:
From just reading the diff (and commit message) it's unclear to me why a former "composite storage" is now equivalent to "no storage".

Reply:
Improved the commit message.